### PR TITLE
Added tests to validate API key

### PR DESCRIPTION
**Pull request by Paul-okpor**
- ### Added more checks for API key 
image

- ### Added clean error response when an unauthorized key is used 
image

- ### Added clean response when paid tier key is used for free 
image

### CODE REVIEW

File: nodejs/tests/api.key.test.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+const apiKey = process.env.ARKHIA_API_KEY;
+const headers = { 'x-api-key': apiKey };
+
+describe('Test to validate API key', () => {
+    test('Should load API Key', () => {
+        expect(apiKey).toBeDefined();
+        expect(apiKey.length).toEqual(32);
+    });
+
+    test('Should validate API Key', async () => {
```
Comment (daniel-bcw) [inline nodejs/tests/api.key.test.js:18]: Also a negative test, saying when doing a request with an `invalid` api Key, it should say unauthorised

Comment (daniel-bcw) [reply nodejs/tests/api.key.test.js:18]: test(`When making a request with invalid ApiKey , should return unauthorised`)

test(`When making a request with no ApiKey, should return `no api key available`)

Comment (Paul-okpor) [reply nodejs/tests/api.key.test.js]: Updated

File: nodejs/tests/api.key.test.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    test('this API Key is not valid', async () => {
+        try {
+            const balancesUrl = `${urlHandler.getApiUrl()}/balances`;
+            const response = await axios.get(balancesUrl, { headers });
+        } catch (error) {
+            expect((error.response.data)).toHaveProperty('status', false);
+            expect(error.response.data.response).toContain('Unauthorized');
+            console.error(error.response.data.response)
+        }
+
```
Comment (daniel-bcw) [inline nodejs/tests/api.key.test.js:27]: Cheeky space here

File: nodejs/tests/api.key.test.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+const axios = require('axios');
+const urlHandler = require('../handlers/url.handler');
+
+
+// Arkhia config
+const apiKey = process.env.ARKHIA_API_KEY;
+const headers = { 'x-api-key': apiKey };
+
+describe('Test to validate API key', () => {
+    test('Should load correct API Key', () => {
```
Comment (daniel-bcw) [inline nodejs/tests/api.key.test.js:13]: `load correct API Key length`

File: nodejs/tests/api.key.test.js [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+        } catch (error) {
+            expect((error.response.data)).toHaveProperty('status', false);
+            expect(error.response.data.response).toContain('Unauthorized');
+            console.error(error.response.data.response)
+        }
+
+
+    });
+
+
```
Comment (daniel-bcw) [inline nodejs/tests/api.key.test.js:31]: Missing 2

1.
Pricing tier > API Key free testing a paid tier
> API Key paid testing a free tier

2. If no api key should return adequate payload

**Review by daniel-bcw — APPROVED** [review]

**Pull request merged**